Headlines/src/DataSources/ProfileDataSource.swift

```diff
@@ -90,49 +92,50 @@ class ProfileDataSource: NSObject, UICollectionViewDataSource {
             cell.label.text = interest.name
             cell.label.textColor = UIColor.black
             cell.backgroundColor = UIColor(white: 240/255.0, alpha: 1)
-            
+
             cell.normalTextColor = UIColor.black
             cell.normalBackgroundColor = UIColor(white: 240/255.0, alpha: 1)
             cell.selectedTextColor = UIColor.black
             cell.selectedBackgroundColor = UIColor(white: 240/255.0, alpha: 1)
-            
+
             return cell
-            
+
         case 1:
             guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "reaction_cell", for: indexPath) as? ProfileReactionCollectionViewCell else {
                 return UICollectionViewCell()
             }
             let reaction = reactions[indexPath.row]
             cell.emojiLabel.text = reaction.reaction
-            
+
             if let n = reaction.news {
                 cell.label.text = n.title
                 cell.imageView.sd_setImage(with: n.imageUrl)
             }
-            
+
             return cell
+
         default:
             return UICollectionViewCell()
         }
     }
-    
+
     func collectionView(_ collectionView: UICollectionView,
                         viewForSupplementaryElementOfKind kind: String,
                         at indexPath: IndexPath) -> UICollectionReusableView {
-        
+
         if let sectionHeader = collectionView.dequeueReusableSupplementaryView(ofKind: kind,
                                                                                withReuseIdentifier: "header",
                                                                                for: indexPath)
             as? SectionLabelHeaderCollectionReusableView {
-            
+
             switch indexPath.section {
             case 0:
                 sectionHeader.label.text = "Intereses"
             case 1:
                 sectionHeader.label.text = "Reacciones"
             default: break
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f5a5ab936ee0d164c160af7374b637afef6eb821/Headlines/src/DataSources/ProfileDataSource.swift#L136"></span>Cases inside a switch should always be on a newline<br />
<code>switch_case_on_newline</code> <code>ProfileDataSource.swift:136</code></td>
    </tr>
  </tbody>
</table>